test/python/transpiler/test_unroller.py

```diff
@@ -535,6 +535,14 @@ def test_unroll_z(self):
         self.ref_circuit.append(U3Gate(0, 0, pi), [0])
         self.compare_dags()
 
+    def test_unroll_snapshot(self):
+        """test unroll snapshot"""
+        num_qubits = self.circuit.num_qubits
+        instr = Snapshot('0', num_qubits=num_qubits)
+        self.circuit.append(instr, range(num_qubits))
+        self.ref_circuit.snapshot('0')
```

Comment:
Does this also need to be `append()` to avoid the conflict with aer's snapshot that was blocking this before?